docs/core/tools/global-tools-how-to-use.md

```diff
@@ -0,0 +1,115 @@
+---
+title: How to install and use a .NET Core global tool
+description: Learn how to install and use a .NET tool as a global tool.
+author: tdykstra
+ms.author: tdykstra
+ms.date: 02/12/2020
+---
+
+# Install and use a .NET Core global tool using the .NET Core CLI
+
+**This article applies to:** ✔️ .NET Core 3.x SDK
```

Comment:
How come the overview applies to 2.1 and the other articles are 3.x?

Reply:
Changed the Overview applies-to text.

Reply:
It should be 2.x and later no? Global tools are supported since earlier. Also, shouldn't the title include the word tutorial in it?

Reply:
You have to have 3.x installed to do the whole series including the local tools tutorial, so it looked strange to me to say "applies to 2.1" right above "prerequisite 3.1".  But I'll change to 2.1+ on all three.